.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Do you have Feature or improvement for Fluent UI React-Native? Let us know.
```

Comment:
To be pedantic (sometimes it's healthy to just admit it) - Should be "Fluent UI React Native" to match the public name used in the README. Otherwise switch to fluentui-react-native to go all in on the repo name